src/Gateway/PayplugGatewayOney3x.php

```diff
@@ -96,6 +96,7 @@ public function get_icon()
             $total_price = floatval(WC()->cart->total);
             $this->oney_response = $this->api->simulate_oney_payment($total_price, 'with_fees');
             $currency = get_woocommerce_currency_symbol(get_option('woocommerce_currency'));
+			$total_price_oney = $total_price + $this->oney_response['x3_with_fees']['nominal_annual_percentage_rate'];
```

Comment:
this won't affect the product page values?

Reply:
no, this is only about showing the installments on the checkout page, not the popup